distribution/osbuild-composer.conf

```diff
@@ -0,0 +1 @@
+u   _osbuild-composer   -   "OSBuild Composer user"
```

Comment:
Why the underscore? I'm probably unaware of some policy :)

Reply:
> It is strongly recommended to pick user and group names that are unlikely to clash with normal users created by the administrator. A good scheme to guarantee this is by prefixing all system and group names with the underscore, and avoiding too generic names.
Source: https://www.freedesktop.org/software/systemd/man/sysusers.d.html#

Reply:
Ah, sounds good. Thanks!